scripts/manageacl.py

```diff
@@ -139,9 +164,42 @@ async def enable_acls(self, endpoint: str):
                         filterable=True,
                     )
                 )
-
+            if not any(field.name == "storageUrl" for field in index_definition.fields):
+                index_definition.fields.append(
+                    SimpleField(
+                        name="storageUrl",
+                        type="Edm.String",
+                        filterable=True,
+                        facetable=True,
+                    ),
+                )
             await search_index_client.create_or_update_index(index_definition)
 
+    async def update_storage_urls(self, search_client: SearchClient):
+        filter = "storageUrl eq ''"
+        documents = await search_client.search("", filter=filter, select=["id", "storageUrl", "oids", "sourcefile"])
+        found_documents = []
+        documents_to_merge = []
+        async for document in documents:
+            found_documents.append(document)
+            if len(document["oids"]) == 1:
+                logger.warning(
+                    "Not updating storage URL of document %s as it has only one oid and may be user uploaded",
```

Comment:
Interesting - we should stop people from doing this? Is there an override, there might be a legitimate use case here

Reply:
Great question! I figured that if they really want to do it, they can change the Python code themselves to do it. I thought about adding an override argument but realized that with the way manageacl.py is structured, any argument becomes a top level instance variable, so that made me less excited to add arguments. I'd be fine with adding --override as well, but I think the usage would be pretty low, since this is only for folks migrating from old repos anyway.